webserver/Pages/Manage/RealEstateCreation.cshtml.cs

```diff
@@ -27,12 +27,7 @@ public class RealEstateCreation : PageModel {
             return Page();
         }
 
-        var realEstate = new RealEstate {
-            Name = Input.Name,
-            Address = Input.Address,
-            Price = Input.Price,
-            CompanyId = company.Id
-        };
+        var realEstate = new RealEstate (0, Input.Name, Input.Address, Input.Price, company.Id);
```

Comment:
It's worse to understand.
Magic numbers are bad. What is the zero thing?

And one per line is better to read.


```suggestion
        int fieldCalledX = 0
        var realEstate = new RealEstate (
          fieldCalledX,
          Input.Name,
          Input.Address,
          Input.Price,
          company.Id
        );
```

Reply:
That field is the Id. I elected to remove it since it's not up to the anybody but .NET itself to assign an Id for the RealEstate